packages/babel-helper-transform-fixture-test-runner/src/index.ts

```diff
@@ -33,14 +33,16 @@ type Module = {
 };
 
 if (!process.env.BABEL_8_BREAKING) {
-  // Introduced in Node.js 8
+  // Introduced in Node.js 10
   if (!assert.rejects) {
     assert.rejects = async function (block, validateError) {
       try {
-        await block();
+        await (typeof block === "function" ? block() : block);
         return Promise.reject(new Error("Promise not rejected"));
       } catch (error) {
-        if (!validateError(error)) {
+        // @ts-ignore Fixme: validateError can be a string | object
+        // see https://nodejs.org/api/assert.html#assertrejectsasyncfn-error-message
+        if (typeof validateError === "function" && !validateError(error)) {
```

Comment:
TS caught some bugs in our polyfill. We can fix this in a separate PR or leave it as-is.

Reply:
It works well enough for what we need :stuck_out_tongue: